templates/SilverStripe/Admin/Includes/LeftAndMain_MenuToggle.ss

```diff
@@ -6,7 +6,7 @@
         </span>
         <span class="cms-help__badge badge badge-info">
                 <% if $CMSVersionNumber %>
-                    <span class="cms-sitename__version" title="$ApplicationName (<%t SilverStripe\Admin\LeftAndMain.Version "Version" %> - $CMSVersion)">$CMSVersionNumber.LimitCharacters(3, '')</span>
+                    <span class="cms-sitename__version" title="$ApplicationName (<%t SilverStripe\Admin\LeftAndMain.Version "Version" %> - $CMSVersion)">$CMSVersionNumber.LimitCharacters(4, '')</span>
```

Comment:
`CMSVersionNumber` will return the full patch release version of CMS. This will display `4.9.` if you are running cms 4.9.x.

I think this other PR will provide a more holistic fix https://github.com/silverstripe/silverstripe-admin/pull/1276

Reply:
True, thanks for the sense-check Max.